### PR TITLE
[python][docs] fix docstring typo

### PR DESCRIPTION
**Pull request by cconvey**
CC: @jcf94

Thanks @octoJon for pointing out the typo!

### CODE REVIEW

File: python/tvm/auto_scheduler/cost_model/xgb_model.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ class XGBModel(PythonBasedModel):
     model_file: Optional[str]
         If is not None, save model to this file after every update.
     adapative_training: bool = False
```
Comment (octoJon) [inline python/tvm/auto_scheduler/cost_model/xgb_model.py:101]: Looks like this one is also 'adapative' instead of 'adaptive'.

Comment (octoJon) [reply python/tvm/auto_scheduler/cost_model/xgb_model.py:101]: Oh, shoot, but the parameter name itself actually has the extra 'a'... Well, sorry for what I thought was just a comment fix which now looks like it fixing spelling within the code.

Comment (cconvey) [reply python/tvm/auto_scheduler/cost_model/xgb_model.py]: PR updated.

**Comment by cconvey** [comment]
@comaniac @merrymercy: Mind giving this a review and (if okay) merge?

File: python/tvm/auto_scheduler/cost_model/xgb_model.py [hunk 2]
```diff
@@ -98,8 +98,8 @@ class XGBModel(PythonBasedModel):
         The random seed
     model_file: Optional[str]
         If is not None, save model to this file after every update.
-    adapative_training: bool = False
-        Whether to use adapatie training, which reduces the training frequency when there are
+    adaptive_training: bool = False
+        Whether to use adapative training, which reduces the training frequency when there are
```
Comment (comaniac) [inline python/tvm/auto_scheduler/cost_model/xgb_model.py:102]: adaptive?

Comment (cconvey) [reply python/tvm/auto_scheduler/cost_model/xgb_model.py:102]: :facepalm:

**Comment by cconvey** [comment]
@comaniac @merrymercy: Would one of you mind giving this a review?

**Comment by comaniac** [comment]
Thanks @cconvey @octoJon

**Pull request merged**